exporter/collector/googlemanagedprometheus/extra_metrics.go

```diff
@@ -113,10 +119,17 @@ func (c Config) addTargetInfoMetric(m pmetric.Metrics) {
 	if !c.ExtraMetricsConfig.EnableTargetInfo {
 		return
 	}
+	ids := make(map[resourceID]bool)
```

Comment:
do you have any preference for `map[resourceId]bool` vs `map[resourceId]struct{}`?

Reply:
ah, looks like struct{} is smaller

Reply:
fixed

Reply:
yeah it's a tiny difference but I thought it could be helpful here